src/test/javascript/spec/component/course/course-overview.component.spec.ts

```diff
@@ -574,14 +578,6 @@ describe('CourseOverviewComponent', () => {
         expect(fixture.nativeElement.querySelector('.container-closed')).toBeNull();
     });
 
-    it('should display course title when navbar is not collapsed', () => {
-        component.isNavbarCollapsed = false;
-        fixture.detectChanges();
-
-        const titleElement = fixture.nativeElement.querySelector('#test-course-title');
-        expect(titleElement.textContent).toContain(component.course?.title);
-    });
-
```

Comment:
Why is this test deleted, I do not see how this is affected by this PR?

Reply:
reverted 👍